reflex/utils/types.py

```diff
@@ -351,6 +355,33 @@ def _issubclass(cls: GenericType, cls_check: GenericType) -> bool:
     if isinstance(cls_base, tuple):
         return False
 
+    # Check that fields of breakpoints match the expected values.
+    if issubclass(cls_base, Breakpoints) and instance is not None:
+
+        def satisfies(cls_check: GenericType):
```

Comment:
If possible we should pull this function out to the top level - otherwise it will get redefined every time we call `issubclass` and may have performance impact